AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -67,6 +67,9 @@ cherry.de,ba-computer.at,stoertebeker.com,inateck.com,biershop-bayern.de,nkd.com
 tweaktown.com##.cookie-prompt
 fandom.com##.wds-banner-notification__container
 660citynews.com##.rdm-daac-notice-template
+projekt-gesund-leben.de##.pixelmate-big-wrapper
+projekt-gesund-leben.de##.pixelmate-revoke
+elopage.com##.consent-management-simple
```

Comment:
Not shown.

Reply:
@BlazDT 
<details><summary> Here it is</summary><img src="https://user-images.githubusercontent.com/18536328/74811750-3789b380-5303-11ea-9973-5feb229531c7.png"></details>

Reply:
That rule is OK, but I suggested exchange of ext-CSS rules for normal CSS rule.

Reply:
There is only one rule with ext-CSS now